ush/templates/FV3.input.yml

```diff
@@ -52,18 +52,24 @@ FV3_HRRR:
     kord_tm: -9
     kord_tr: 9
     kord_wz: 9
+    nord_tr: 2
     nrows_blend: 10
     regional_bcs_from_gsi: false
     write_restart_with_bcs: false
   gfs_physics_nml:
     <<: *gsd_sar_phys
     cdmbgwd: [3.5, 1.0]
     do_mynnsfclay: true
+    do_sfcperts: !!python/none
```

Comment:
Currently we have do_sfcperts = .false. Do you want to remove it?

Reply:
This line sets it to the default. I can't seem to find anywhere in the model that it's actually used or set! I was looking for the default value, but that doesn't seem to be straightforward.

I do see that it's a flag for stochastic surface perturbations option. Maybe we'd have had to turn on stochastic perturbations for it to matter, anyway.

@JeffBeck-NOAA, do you have any idea what this should be? And what the default is?

Reply:
@christinaholtNOAA, !!python/none removes the entry from the resulting namelist, correct?  This then results in whatever the default value is set to in the dycore.  The default value for do_sfcperts is false, so having it set to !!python/none or .false. will do the same thing.  Note that this entry has been replaced by lndp_type in the new model code.

Reply:
@JeffBeck-NOAA, that is correct. We are not setting this value, so I think that's fine.

Do we need to be setting `lndp_type` to a particular value, or will it just be the equivalent of the old "false" by default?